.github/workflows/build.yml

```diff
@@ -22,18 +22,19 @@ jobs:
           java-version: ${{ matrix.java }}
           architecture: ${{ matrix.architecture }}
 
-      - name: Build the Native Image
-        run: mvn package && mvn clean package -Pnative -Dnative-image.docker-build=true
+      - name: Maven Build
+        run: mvn package
 
       - name: Build the Docker Image
-        run: docker build . -t hazelcast-guides/hazelcast-quarkus-native
+        run: docker build . -f Dockerfile -t hazelcast-guides/hazelcast-quarkus-native
```

Comment:
I added `-f Dockerfile` on purpose to make the difference between `Dockerfile.native` explicit